src/sidebar/services/groups.js

```diff
@@ -50,6 +50,12 @@ function groups($rootScope, store, api, isSidebar, localStorage, serviceUrl, ses
   function filterGroups(groups, isLoggedIn, directLinkedAnnotationId) {
     // Logged-in users always see the "Public" group.
     if (isLoggedIn) {
+
```

Comment:
It doesn't actually matter for the LMS use case but I think it would be more consistent to respect the `groups` property regardless of whether the user is logged-in or not. Since `services[index].groups` only applies when third-party accounts are being used, the "Public" group will not be in the returned list.